js/main.js

```diff
@@ -1411,7 +1453,12 @@ class App extends React.Component {
               isFocused={pane.id === this.state.focusedPaneID}
               w={panelayout.w}
               h={panelayout.h}
-              appApi={{ sendPaneMessage: this.sendPaneMessage }}
+              width={this.w2p(panelayout.w)}
+              height={this.h2p(panelayout.h) - 14}
```

Comment:
At some point this magic number should end up in a constant

Reply:
Moved it to a constant for now. Ideally the css layout will be done in a way in the future where this won't be needed.